docs/build.md

```diff
@@ -82,13 +82,21 @@ Distribution packages are created by exec `grunt build` on Linux platform (e.g.
 
 After installing the supported version of `node` and `npm`, install build dependency packages.
 
-Ubuntu/Debian:
+**Ubuntu/Debian:**
+
+```
+$ npm i -g grunt-electron-installer-debian
+```
 
 ```
 $ sudo apt-get install -y rpm fakeroot
 ```
 
-Fedora:
+**Fedora:**
+
+```
+$ npm i -g grunt-electron-installer-redhat
```

Comment:
I believe both distributions still need both electron installer packages (`grunt-electron-installer-debian` and `grunt-electron-installer-redhat`) for `grunt build` to work properly. If either package is missing, then `grunt build` will fail with an error, and prompt the user to re-run the command with the `--force` flag. I think we should refactor this in one of two ways:
* instruct both Ubuntu/Debian and Fedora users to install both grunt-electron-installer packages
* instruct both Ubuntu/Debian and Fedora users to install the respecitve electron-installer package, and instruct them to run `grunt build --force` instead. I'm not keen on this approach because instructing users to always run commands with the --force flag feels like a dangerous habit to form.

Reply:
**If possible** we can also change the gruntfile to require only the package that is needed for that specific build.

But, I'll proceed with both dependency package install.